application/config/rest/v1/survey.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+use \LimeSurvey\Api\Command\V1\{
+    SurveyList,
+    SurveyDetail,
+    SurveyPatch
+};
+use LimeSurvey\Api\Rest\V1\SchemaFactory\{
+    SchemaFactoryError,
+    SchemaFactorySurveyList,
+    SchemaFactorySurveyDetail,
+    SchemaFactorySurveyPatch
+};
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// REST V2 Survey Config
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+$errorSchema = (new SchemaFactoryError)->create();
+$surveyPatchSchema = (new SchemaFactorySurveyPatch)->create();
+
+$rest = [];
+
+$rest['v1/survey'] = [
+    'GET' => [
```

Comment:
Do we have a description which changes are needed to add a new API endpoint in current design?

Reply:
Not yet.

Reply:
It is not written as a step by step guide but this document has the information: https://writer.zoho.eu/writer/open/afa6w6e4df6bf98bc4d86bcbb290ee6e13299

Reply:
https://manual.limesurvey.org/REST_API

Needs to be written in the manual too, for outside contributors to follow, but we can wait until after it's merged, to avoid any potential wasted effort.